index.html

```diff
@@ -14,12 +15,28 @@
   <![endif]-->
 </head>
 
-<body>
-
-  
-     
+<body>  
   <div class="container home">
-   
+      <section class = "navsection">
+        <div class = "maintitle">
+          <h1>Fun Bus</h1>
```

Comment:
Nothing wrong here, but normally you'd want a `div` to manipulate multiple elements inside div, but since there's only the h1, you could just remove the div there.